docs/reference/aggregations/metrics/cardinality-aggregation.asciidoc

```diff
@@ -150,10 +150,15 @@ public static void main(String[] args) {
 
 image:images/cardinality_error.png[]
 
-For all 3 thresholds, counts have been accurate up to the configured threshold
-(although not guaranteed, this is likely to be the case). Please also note that
-even with a threshold as low as 100, the error remains very low, even when
-counting millions of items.
+For all 3 thresholds, counts have been accurate up to the configured threshold.
+Although not guaranteed, this is likely to be the case.  Accuracy in practice depends
+on the dataset in question.  Because the HyperLogLog++ algorithm depends on the
+leading zeros of hashed values, the exact distributions of hashes in a dataset
+can affect how accurate the cardinality is.  In general, most datasets show
+consistently good accuracy.
+
+Please also note that even with a threshold as low as 100, the error remains
+very low, even when counting millions of items.
```

Comment:
This bit has always bothered me a bit because "remains very low" is not quantitive. Happy to leave this for now since its not changed in the PR but I wonder if we should quantify this after this sentence by referencing a benchmark?

Reply:
Hm, well based on the chart (which is based on the test code in the comments of the doc) it looks like the error is 1-6%:

![image](https://user-images.githubusercontent.com/1224228/47231900-2db51080-d39c-11e8-9d66-41030eee7d65.png)

Maybe we could just replace the wording with those numbers?  I think the chart was added some time after the paragraph was written

Reply:
I'm fine with adding the results of the benchmark here since we already have it